### DISCUSSION TITLE
Title not displayed for 30-minute events in day grid, any ideas?

### DISCUSSION THREAD

**Discussion by jmerrow** [vkurko/calendar#186] [Q&A] [answered]
Thanks for the great project. Is there any way to either display the title on the top line next to the start and end times, or increase the vertical granularity of the grid say to 15 minutes? I need a way for titles to be visible for 30-minute events in day grid view.

Thanks,
Josh Merrow

**Comment by vkurko** [comment, marked as answer]
Hello, please take a look at the [slotDuration](https://github.com/vkurko/calendar#slotduration) option.

**Reply by jmerrow** [reply]
Oops, missed that. Perfect, thanks!
